### PR TITLE
Update k8s license docs to account for license autoload

### PR DESCRIPTION
**Pull request by thisisnotashwin**

### CODE REVIEW

File: website/content/docs/k8s/installation/deployment-configurations/consul-enterprise.mdx [hunk 1]
```diff
@@ -39,6 +39,19 @@ server:
...
+```yaml
+# config.yaml
+global:
+  image: 'hashicorp/consul-enterprise:1.4.3-ent'
+server:
+  enterpriseLicense:
+    secretName: 'consul-ent-license'
+    secretKey: 'key'
+    enableLicenseAutoload: false
+```
```
Comment (lkysow) [inline website/content/docs/k8s/installation/deployment-configurations/consul-enterprise.mdx:53]: Maybe add a NOTE that `enableLicenseAutoload` must be set to `false`.

File: website/content/docs/k8s/installation/deployment-configurations/consul-enterprise.mdx [hunk 2]
```diff
@@ -27,7 +27,7 @@ global:
   image: 'hashicorp/consul-enterprise:1.4.3-ent'
 ```
 
-Add the name and key of the secret you just created to `server.enterpriseLicense`.
+Add the name and key of the secret you just created to `server.enterpriseLicense`, if using Consul version 1.10+.
```
Comment (lkysow) [inline website/content/docs/k8s/installation/deployment-configurations/consul-enterprise.mdx:30]: I'd change the `global.image` below to be `1.10` to make that clear.

**Review by lkysow — APPROVED** [review]

File: website/content/docs/k8s/installation/deployment-configurations/consul-enterprise.mdx (hunk 2 above)
Comment (thisisnotashwin) [reply website/content/docs/k8s/installation/deployment-configurations/consul-enterprise.mdx:30]: Good catch!

**Review by ishustava — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/388303.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 325b18ad07a4edb4d4cc7c36f9b1c26c81cf2a08 onto `stable-website` succeeded!

**Pull request merged**